backend/routes/post.py

```
@@ -75,6 +79,59 @@ def get_post_additional_data(post: Post, db: Session):
+
+def get_post_by_id(db: Session, post_id: int, user_id: int = None):
+    query = db.query(Post).filter(Post.id == post_id)
+    if user_id is not None:
+        query = query.filter(Post.user_id == user_id)
+    post = query.first()
+    if not post:
+        raise HTTPException(status_code=404, detail=STATUS_404_ERROR)
+    return post
+
+
+def update_post_content(post: Post, content: Optional[str]):
```

Ruff (0.8.2)

Missing return type annotation for public function `update_post_content`

Add return type annotation: `None`

(ANN201)